<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Collect</class>
 <widget class="QWidget" name="Collect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Collect</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_sidebar">
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>flechazo</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_sidebar_info">
        <property name="title">
         <string>GroupBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_sidebar_log">
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>120</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_collect">
     <property name="title">
      <string>收藏</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_collect_config">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="title">
         <string>配置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_collect_config_path"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_collect_config_open">
           <property name="text">
            <string>打开路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_collect_config_flush">
           <property name="text">
            <string>读取刷新</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_collect_list">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_collect_list_pursuit">
         <attribute name="title">
          <string>人生事业</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_collect_list_study">
         <attribute name="title">
          <string>终身学习</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_collect_list_life">
         <attribute name="title">
          <string>认真生活</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_collect_list_love">
         <attribute name="title">
          <string>规划爱情</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_collect_list_collect">
         <attribute name="title">
          <string>收藏</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_collect_list_website">
         <attribute name="title">
          <string>收藏网站</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_collect_operate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="title">
         <string>操作</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_collect_operate_add">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_collect_operate_delete">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_collect_operate_openlocal">
           <property name="text">
            <string>打开本地仓库</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_collect_operate_openremote">
           <property name="text">
            <string>打开远程仓库</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
